src/frontend/src/components/public/ProgressBar/Stage.tsx

```diff
@@ -1,17 +1,21 @@
-import { ReactElement } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { ReactElement } from 'react'
 import { StageProps } from './interfaces';
 
-const Stage = ({ id, label, icon, active }: StageProps): ReactElement => {
-  return (
-    <div className={`stage-block ${active ? 'active' : ''}`}>
-      <div className="circle">
-        <div className="inner-circle">
-          <img src={icon} />
-        </div>
-      </div>
-      <p className="stage-label">{label}</p>
-    </div>
-  );
-};
+const Stage = ({id, label, icon, active}: StageProps): ReactElement => {
+    return (
+        <>
+            <div className={`stage-lead-line ${active ? "active" : ""}`}></div>
+            <div className={`stage-block`}>
+                <div className={`circle ${active ? "active" : ""}`}>
+                    <div className="inner-circle">
+                        <FontAwesomeIcon icon={icon} />
+                    </div>
+                </div>
+                <p className="stage-label">{label}</p>
+            </div>
+        </>
```

Comment:
Looks like you have taken latest `main` and still indentation seems odd - 4 spaces instead of 2.

Reply:
Should husky be fixing this? Or should we set up a .prettierrc to format on save?

Reply:
Ideally prettierrc on save; But that need to be setup locally. Even if that's not setup, husky shouldn't have allowed you to commit - and thrown the Linting error when you do `git commit `

Reply:
Hmmm, I definitely didn't get any errors.

Reply:
Am I okay to merge? I do want to resolve this formatting issue for future PRs though.

Reply:
Yeh, let's go ahead. But it's just that whoever merges next and if the linting works then - will have to get the fix the changes before proceeding. We should. be good for now. Let's go ahead 👍